### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: src/Cache/SchemaCache.php [hunk 1]
```diff
@@ -38,14 +45,24 @@ public function getDuration(): int
 
     public function isExclude(string $value): bool
```
Comment (samdark) [inline src/Cache/SchemaCache.php:46]:
```suggestion
    public function isExcluded(string $value): bool
```

File: src/Cache/SchemaCache.php [hunk 2]
```diff
@@ -38,14 +45,24 @@ public function getDuration(): int
...
     }
 
     public function isEnabled(): bool
     {
         return $this->enabled;
     }
 
+    public function set($key, $value, $ttl = null, Dependency $dependency = null): bool
+    {
+        return $this->cache->set($key, $value, $ttl, $dependency);
```
Comment (samdark) [inline src/Cache/SchemaCache.php:63]: It goes against `Psr\SimpleCache\CacheInterface` so `Yiisoft\Cache\CacheInterface` should be used in this class.

File: src/Cache/SchemaCache.php [hunk 3]
```diff
@@ -6,6 +6,8 @@
 
 use Psr\SimpleCache\CacheInterface;
 use Yiisoft\Cache\CacheKeyNormalizer;
+use Yiisoft\Cache\Dependency\Dependency;
+use Yiisoft\Cache\Dependency\TagDependency;
 
 final class SchemaCache
```
Comment (samdark) [inline src/Cache/SchemaCache.php:12]: Need docs overall about this class and its methods. Some aren't clear enough without docs.

**Review by samdark — CHANGES_REQUESTED** [review]

File: src/Cache/SchemaCache.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Cache/SchemaCache.php]: done.

File: src/Cache/SchemaCache.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Cache/SchemaCache.php:104]: done.

File: src/Cache/SchemaCache.php (hunk 3 above)
Comment (terabytesoftw) [reply src/Cache/SchemaCache.php:15]: done,

File: src/Cache/QueryCache.php [hunk 4]
```diff
@@ -26,16 +29,37 @@ public function __construct(CacheInterface $cache, CacheKeyNormalizer $keyNormal
...
+     * then the key will be returned back as it is, integers will be converted to strings. Otherwise,
+     * a normalized key is generated by serializing the given key and applying MD5 hashing.
+     *
+     * @param mixed $key The key to be normalized.
+     *
+     * @return string The normalized cache key.
+     */
     public function normalize($key): string
     {
         return $this->keyNormalizer->normalize($key);
```
Comment (samdark) [inline src/Cache/QueryCache.php:45]: We're going to break it again soon :)

**Review by samdark — APPROVED** [review]

**Pull request merged**